custom-connectors/CopilotForFinanceCommunications/apiDefinition.swagger.json

```diff
@@ -1,765 +1,1376 @@
-                  "x-ms-summary": "Next Link"
                 }
               }
             }
           },
-          "401": {
-            "description": "Unauthorized"
-          },
-          "403": {
-            "description": "Forbidden"
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
           }
-        },
-        "summary": "List chat thread messages",
-        "description": "List messages From chat thread.",
-        "operationId": "ListMessages",
+        }
+      }
+    },
+    "/contacts/create": {
+      "post": {
+        "tags": [
+          "Contacts"
+        ],
+        "summary": "Creates contact",
+        "description": "Creates contact.",
+        "operationId": "Contacts_create",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
         "parameters": [
           {
-            "$ref": "#/parameters/accessToken"
-          },
-          {
-            "$ref": "#/parameters/chatThreadId"
-          },
-          {
-            "$ref": "#/parameters/apiVersion"
+            "in": "header",
+            "name": "x-copilotforfinace-api-version",
+            "description": "Specifies the API version.",
+            "required": true,
+            "type": "string"
           },
           {
-            "$ref": "#/parameters/listMessagesStartTime"
+            "in": "query",
+            "name": "language",
+            "description": "The language code is in BCP 47 format.",
+            "type": "string"
           },
           {
-            "$ref": "#/parameters/listMessagesMaxPageSize"
+            "in": "body",
+            "name": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ContactForUpdate"
+            }
           }
-        ]
-      },
-      "post": {
+        ],
         "responses": {
           "201": {
-            "description": "Success",
+            "description": "Creates the contact and returns the contact object.",
             "schema": {
-              "type": "object",
-              "properties": {
-                "id": {
-                  "type": "string",
-                  "description": "id"
-                }
-              }
+              "$ref": "#/definitions/ContactForUpdateResponse"
             }
           },
-          "401": {
-            "description": "Unauthorized"
-          },
-          "403": {
-            "description": "Forbidden"
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
           }
-        },
-        "summary": "Send message to chat thread",
-        "description": "Send message to chat thread.",
-        "operationId": "SendChat",
```

Comment:
The operation has been deleted. This change breaks existing flows. ValuePath: operation/SendChat

Reply:
We are adding a new template for a custom connector, so nothing is actually deleted.
Endpoint is deleted compared to prior iteration because prior iteration had incorrect swagger included.